examples/images_contours_and_fields/image_antialiasing.py

```diff
@@ -5,12 +5,21 @@
 
 Images are represented by discrete pixels, either on the screen or in an
 image file.  When data that makes up the image has a different resolution
-than its representation on the screen we will see aliasing effects.
+than its representation on the screen we will see aliasing effects.  How
+noticeable these are depends on how much down-sampling takes place in
+the change of resolution (if any)
 
-The default image interpolation in Matplotlib is 'antialiased'.  This uses a
-hanning interpolation for reduced aliasing in most situations. Only when there
-is upsampling by a factor of 1, 2 or >=3 is 'nearest' neighbor interpolation
-used.
+When subsampling data, aliasing is reduced by smoothing first and then
+subsampling the smoothed data.  In Matplotlib, we can do that
+smoothing before mapping the data to colors, or we can do the smoothing
+on the RGB(A) data in the final image.  The difference between these is
+shown below, and controlled with the *interp_space* keyword argument.
```

Comment:
I am not a huge fan of this name, I think of this more as changing the order of the pipeline rather than changing the space of the interpolation.  I agree it does change the space, but the re-ordering (from resample -> norm -> colormap to norm -> colormap -> resample) is the key difference.

Reply:
I'm not wed to the name, which was suggested by @anntzer, though I think its pretty good: `interp_stage`?  `interp_step`?